fuse/data/ops/ops_common.py

```diff
@@ -487,3 +487,25 @@ class OpZScoreNorm(OpBase):
     def __call__(self, sample_dict: NDict, key: str, mean: float, std: float):
         sample_dict[key] = (sample_dict[key] - mean) / std
         return sample_dict
+
+
+class OpCond(OpBase):
+    """Apply given op if the condition (either directly specified or read from the sample_dict) is True"""
```

Comment:
Currently, we support two options.
Getting a boolean directly or reading it from sample_dict.
It means that if some logic is required - it can be done in previous ops and store in sample_dict.
We can support callback of course - in the first use case that requires it.